### PR TITLE
Quote previous reply

### PR DESCRIPTION
**Pull request by Kharchevskyi**
close https://github.com/FlowCrypt/flowcrypt-ios/issues/146

### CODE REVIEW

File: FlowCrypt/Controllers/Compose/ComposeDecorator.swift [hunk 1]
```diff
@@ -59,6 +60,34 @@ struct ComposeDecorator: ComposeDecoratorType {
...
+
+        let date = dateFormatter.string(from: info.sentDate)
+
+        dateFormatter.dateStyle = .none
+        dateFormatter.timeStyle = .short
+        let time = dateFormatter.string(from: info.sentDate)
+
+        let from = info.recipient?.displayName
+            ?? info.recipient?.mailbox
+            ?? "no subject"
```
Comment (tomholub) [inline FlowCrypt/Controllers/Compose/ComposeDecorator.swift]: should say `"unknown sender"`

File: FlowCrypt/Controllers/Compose/ComposeDecorator.swift [hunk 2]
```diff
@@ -59,6 +60,34 @@ struct ComposeDecorator: ComposeDecoratorType {
...
+        dateFormatter.dateStyle = .short
+        dateFormatter.timeStyle = .none
+
+        let date = dateFormatter.string(from: info.sentDate)
+
+        dateFormatter.dateStyle = .none
+        dateFormatter.timeStyle = .short
+        let time = dateFormatter.string(from: info.sentDate)
+
+        let from = info.recipient?.displayName
```
Comment (tomholub) [inline FlowCrypt/Controllers/Compose/ComposeDecorator.swift]: No need to use display name, just go with email

File: FlowCrypt/Resources/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -35,6 +35,8 @@
 "compose_no_pub_sender" = "Missing sender public key. Is FlowCrypt iOS app well set up?";
 "compose_error" = "Could not compose message";
 "compose_reply_successfull" = "Reply successfully sent";
+"compose_reply_from_title" = "Reply from iOS";
+"compose_reply_from" = "On %@ at %@ %@ wrote:"; // Date, time, subject
```
Comment (tomholub) [inline FlowCrypt/Resources/en.lproj/Localizable.strings]: Date, time, **sender**

File: FlowCrypt/Controllers/Compose/ComposeDecorator.swift [hunk 4]
```diff
@@ -59,6 +60,34 @@ struct ComposeDecorator: ComposeDecoratorType {
...
+            ?? info.recipient?.mailbox
+            ?? "no subject"
+
+        let text: String = "\n\n"
+            + "compose_reply_from_title".localized
```
Comment (tomholub) [inline FlowCrypt/Controllers/Compose/ComposeDecorator.swift]: Don't add this title, it's not needed

File: FlowCrypt/Controllers/Compose/ComposeDecorator.swift [hunk 5]
```diff
@@ -59,6 +60,34 @@ struct ComposeDecorator: ComposeDecoratorType {
...
+            + "\n"
+            + "compose_reply_from".localizeWithArguments(date, time, from)
+            + "\n\n"
```
Comment (tomholub) [inline FlowCrypt/Controllers/Compose/ComposeDecorator.swift]: I think it should be just one newline, not two

File: FlowCrypt/Controllers/Compose/ComposeDecorator.swift (hunk 4 above)
Comment (Kharchevskyi) [reply FlowCrypt/Controllers/Compose/ComposeDecorator.swift]: `Reply from iOS` is not needed? It was in task description. Removed

Comment (tomholub) [reply FlowCrypt/Controllers/Compose/ComposeDecorator.swift]: That was supposed to be example of a message reply that the user is sending.

If I receive a message `This was the original message\nNow it is quoted`, and I type type `reply from iOS`, then the sent message should be exactly as defined in the issue:
```
reply from iOS

On 2019-11-21 at 12:11, mindoutofframe@gmail.com wrote:
> This was the original message
> Now it is quoted
```

**Pull request merged**